### DISCUSSION TITLE
How to cite?

### DISCUSSION THREAD

**Discussion by rgaiacs** [Project-MONAI/MONAILabel#255] [General]
What is the best way to give credit to MONAI Label in an academic paper? Is there a manuscript in https://arxiv.org/ or another pre-print server?

**Comment by aihsani** [comment]
Hi @rgaiacs, thank you for your interest. You will be able to add a citation for MONAI Label in the near future, however, for now it should suffice to state that you have used MONAI Label for your purposes and perhaps add a link to the repo in the footer of the page.

**Comment by tvercaut** [comment]
To align with the practice in MONAI (core), until we have a scientific paper to cite, we could have the tagged version of MONAI Label hosted on zenodo and use the associated doi for citation purposes. See e.g.:
https://github.com/Project-MONAI/MONAI/pull/2663

**Comment by SachidanandAlle** [comment]
https://github.com/Project-MONAI/MONAILabel/blob/main/CITATION.cff
